apps/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "yarn lint:scss && yarn lint:js",
     "lint:fix": "yarn lint:scss --fix && yarn lint:js --fix",
-    "lint:js": "eslint --format ./.eslintCustomMessagesFormatter.js --ext .js,.jsx,.ts,.tsx src test --resolve-plugins-relative-to .",
+    "lint:js": "eslint --format ./.eslintCustomMessagesFormatter.js --ext=.js,.jsx,.ts,.tsx .",
```

Comment:
`yarn lint:js` now applies to all of /apps, relying on /apps/.eslintignore for skipping files. Previously eslint was just applied to /apps/src and /apps/test, which resulted in it not flagging some files on `yarn lint` that WERE flagged by the pre-commit hook.